README.md

```diff
@@ -76,6 +93,20 @@ and
 [NOTICE](https://github.com/InsightSoftwareConsortium/ITK/blob/master/NOTICE)
 files for details.
 
+Supporting ITK
+--------------
```

Comment:
Thinking out loud, we should also add a paragraph around here related to how to obtain commercial support from Kitware, but that can come in a follow-up PR.

Reply:
Sounds fair to me. A separate PR would be best IMHO.